torch/ao/quantization/fx/_model_report/detector.py

```diff
@@ -1051,18 +1068,22 @@ def determine_observer_insert_points(self, prepared_fx_model: GraphModule) -> Di
 
         return obs_fqn_to_info
 
-    def _calculate_outlier_info(self, percentile_ratios: torch.Tensor, counted_batches: torch.Tensor) -> Dict[str, List[bool]]:
+    def _calculate_outlier_info(
+            self, percentile_ratios: torch.Tensor, counted_batches: torch.Tensor, total_batches: int
+    ) -> Dict[str, List[bool]]:
         r"""
         Gives info on whether the percentile ratios cacluated would be considered outliers
         Also gives information on whether the collected data is statistically significant to make this claim
 
         Args:
             percentile_ratios (torch.Tensor): The average percentile_ratios per channel calculated by the observer
             counted_batches (torch.Tensor): The number of batches used for average calculation per tensor
+            total_batches (int): The total number of batches that passed through observer in this epoch
 
         Returns a dictionary mapping:
             "outliers_detected" : list of bools per channel that are true if it is considered an outlier
-            "above_sample_threshold_count": if the per channel calculation had at least 30 samples (a random threshold)
+            "sufficient_batches": if o_r was >= statistical_threshold:
```

Comment:
This value is actually a fraction but `sufficient_batches` still makes it sound like the unit is number of batches. I feel you need "ratio" or "fraction" in there somewhere.